.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
+
+[*]
+end_of_line = crlf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.xml]
+indent_style = space
+
+[*.cs]
+indent_style = space
+indent_size = 4
+csharp_new_line_before_open_brace = properties,control_blocks,types,methods,anonymous_types,object_collection_array_initalizers
```

Comment:
I would drop everything `charp_*` as this is subjective and should be discussed and submitted as a separate PR.